postgresql-async/src/test/java/com/github/aysnc/sql/db/postgresql/QuerySpec.kt

```diff
@@ -27,4 +29,21 @@ class QuerySpec : DatabaseTestHelper() {
         }
     }
 
+    @Test
+    fun `connection should be able to namespaced to a schema`() {
+        withHandler(ContainerHelper.defaultConfiguration.copy(currentSchema = "testing")) { handler ->
+            executeDdl(handler, this.messagesCreate)
+            executeDdl(handler, "INSERT INTO messages (content) VALUES ('Hello')")
+        }
+
+        withHandler { handler ->
+            val rows = executePreparedStatement(handler, "SELECT id FROM messages").rows
+            assertThat(rows.length).isEqualTo(1)
+
+            verifyException(ExecutionException::class.java) {
```

Comment:
I don't think this is the right exception but I'm not sure what to do for it.

Reply:
you can do something like this to check the inner exception: verifyException(ExecutionException::class.java, IllegalStateException::class.java)

Reply:
I'm not sure what the inner exception would be so best to just leave as is for now.